protocol/contact.go

```diff
@@ -55,6 +56,14 @@ func (c *Contact) CanonicalImage(profilePicturesVisibility accounts.ProfilePictu
 	return c.Identicon
 }
 
+type VerificationStatus int
+
+const (
+	VerificationStatusUNVERIFIED VerificationStatus = 0
+	VerificationStatusVERIFYING  VerificationStatus = 1
+	VerificationStatusVERIFIED   VerificationStatus = 2
+)
```

Comment:
Would you consider using `iota`, it seems like a nice fit for your enums

```suggestion
const (
	VerificationStatusUNVERIFIED VerificationStatus = iota
	VerificationStatusVERIFYING  VerificationStatus
	VerificationStatusVERIFIED   VerificationStatus
)
```